src/main/java/uk/gov/companieshouse/company/profile/controller/CompanyProfileController.java

```diff
@@ -94,4 +97,23 @@ public ResponseEntity<Void> deleteExemptionsLink(
         companyProfileService.deleteExemptionsLink(contextId, companyNumber);
         return ResponseEntity.status(HttpStatus.OK).build();
     }
+
+    /**
+     * Add an officers link to a company profile for the given company number.
+     *
+     * @param companyNumber The number of the company
+     * @return no response
+     */
+    @PatchMapping("/company/{company_number}/links/officers")
+    public ResponseEntity<Void> addOfficersLink(
+            @RequestHeader("x-request-id") String contextId,
+            @PathVariable("company_number") String companyNumber) {
+        logger.info(String.format("Payload successfully received on PATCH endpoint "
+                + "with contextId %s and company number %s", contextId, companyNumber));
+
+        String linkType = "officers";
+        String deltaType = "officer_delta";
+        companyProfileService.addOfficersLink(contextId, companyNumber, linkType, deltaType);
```

Comment:
The parameters for addOfficersLink could be encapsulated within some kind of link request object.